CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # buildevents changelog
 
+## 0.16.0 - 2024-02-29
+
+### Enhancements
+
+- upgrade to libhoney-go 1.21.0 to support classic ingest keys (#208) | [@cewkrupa](https://github.com/cewkrupa)
+
+### Maintenance
+
+- Bump github.com/honeycombio/beeline-go from 1.13.0 to 1.14.0 (#206) | [dependabot](https://github.com/dependabot)
```

Comment:
Just did a quick check to ensure we're only using the beeline for the [propagation.MarshalHoneycombTraceContext](https://github.com/honeycombio/beeline-go/blob/834e44f3d6cc85d85805dff628744c4fb2b09ee9/propagation/honeycomb.go#L38) method which doesn't look at key lengths at all.

Said differently: we don't need to wait on the updated beeline-go here